### ISSUE TITLE
Cannot connect to ratbagd 

### ISSUE THREAD

**Issue by danrobi11** [flathub/org.freedesktop.Piper#8]
I get this error when I lauch piper: "Cannot connect to ratbagd. Please make sure ratbagd is running and your user is in the required group"

Piper v0.4
flatpak v1.2.5
MX-Linux.19.2 (patito feo)
Debian GNU/Linux 10 (buster)
Linux 4.19.0-9-amd64 x86_64
SysV init version: 2.93

**Comment by jmariondev** [comment]
Do you have `ratbagd` running outside of the flatpak environment? It looks like this flatpak is intended to talk to an existing ratbagd instance listening on the system dbus `org.freedesktop.ratbag1`. It looks like this is provided by the [ratbagd](https://packages.debian.org/buster/ratbagd) package on Debian buster. (It was part of [libratbag](https://archlinux.org/packages/community/x86_64/libratbag/) for me on Arch, FWIW).

**Comment by whot** [comment]
Yes, this is indeed the case - unfortunately ratbagd required permissions to access the devices that aren't available inside a flatpak, so we can't ship that as a flatpak and needs to be installed from the system repositories.
